conftest.py

```diff
@@ -0,0 +1,29 @@
+from matrix import Matrix
+import pytest
+
+@pytest.fixture
+def matrix_one() -> Matrix:
```

Comment:
Do not forget to write the docstrings
```suggestion
def matrix_one() -> Matrix:
"""Fixture to create a ``Matrix`` instance."""
```

Reply:
I wrote them